components/button/button-icon.js

```diff
@@ -62,28 +62,28 @@ class ButtonIcon extends SlottedIconMixin(PropertyRequiredMixin(ThemeMixin(Butto
 					display: none;
 				}
 				:host([translucent]) {
-					--d2l-button-icon-background-color: rgba(0, 0, 0, 0.5);
-					--d2l-button-icon-background-color-hover: var(--d2l-color-celestine);
+					--d2l-button-icon-background-color-default: rgba(0, 0, 0, 0.5);
+					--d2l-button-icon-background-color-hover-default: var(--d2l-color-celestine);
 					--d2l-focus-ring-color: white;
 					--d2l-focus-ring-offset: -4px;
 					--d2l-button-icon-fill-color: white;
 					--d2l-button-icon-fill-color-hover: white;
 				}
 				:host([theme="dark"]) {
-					--d2l-button-icon-background-color: transparent;
-					--d2l-button-icon-background-color-hover: rgba(51, 53, 54, 0.9); /* tungsten @70% @90% */
+					--d2l-button-icon-background-color-default: transparent;
+					--d2l-button-icon-background-color-hover-default: rgba(51, 53, 54, 0.9); /* tungsten @70% @90% */
```

Comment:
The implication here is that consumers overriding the background colour now need to be aware of dark mode. I guess this will only be temporary anyway until we tackle that?

Reply:
Oh yeah, list will need to provide a different override color when in dark mode, or maybe we'll identify a semantic color for that override when the time comes.